nixos/modules/security/acme.nix

```diff
@@ -222,10 +231,14 @@ let
       serviceConfig = commonServiceConfig // {
         Group = data.group;
 
-        # AccountDir dir will be created by tmpfiles to ensure correct permissions
-        # And to avoid deletion during systemctl clean
-        # acme/.lego/${cert} is listed so that it is deleted during systemctl clean
-        StateDirectory = "acme/${cert} acme/.lego/${cert} acme/.lego/${cert}/${certDir}";
+        # Keep in mind that these directories will be deleted if the user runs
+        # systemctl clean --what=state
+        # acme/.lego/${cert} is listed for this reason.
+        StateDirectory =
```

Comment:
(nitpick) I think a list of string would have the same effect.

Reply:
This has the effect of changing the generated entries in the unit config from a single line of space separated values to multiple lines for each value. The resulting behaviour is the same, so I changed it anyway in favour of the Nix syntax improvement.